packages/snaps-execution-environments/src/common/lockdown/lockdown-events.test.browser.ts

```diff
@@ -0,0 +1,25 @@
+// eslint-disable-next-line import/unambiguous
+import { expect } from '@wdio/globals';
+
+import { executeLockdownEvents } from './lockdown-events';
+
+describe('lockdown events security', () => {
+  it('should lockdown events and made event properties inaccessible', async () => {
+    executeLockdownEvents();
+
+    let resolvePromise: (value: unknown) => void;
+    const promise = new Promise((resolve) => {
+      resolvePromise = resolve;
+    });
+
+    const eventTarget = new EventTarget();
+    eventTarget.addEventListener('just-test-event', (eventObject) => {
+      // eslint-disable-next-line @typescript-eslint/unbound-method
+      expect(eventObject.composedPath).toBeUndefined();
```

Comment:
Can we add `expect.assertions(1)` at the top of the test?

Reply:
Webdriverio's `expect` is similar to Jest, but doesn't have all the Jest features. You can get around it like this, which guarantees that the `expect` is called.

```ts
const eventTarget = new EventTarget();

const promise = new Promise((resolve) => {
  eventTarget.addEventListener('just-test-event', (eventObject) => {
    // eslint-disable-next-line @typescript-eslint/unbound-method
    resolve(eventObject.composedPath);
  });
});

const testEvent = new Event('just-test-event');
eventTarget.dispatchEvent(testEvent);

const result = await promise;
expect(result).toBeUndefined();
```